apiary.apib

```diff
@@ -8443,6 +8443,368 @@ Survey response on BirdEye platform can be submitted using:
 
 + Response 200
 
++ Response 400
+
+  [1315][]
+
++ Response 401
+
+  [1161][]
+
++ Response 401
+
+  [1167][]
+
++ Response 404
+
+  [1011][]
+
+## List Responses [/survey/ext/list/responses/{survey_id}{?api_key,businessNumber,page,size,sortby,sorder}]
+Survey responses for a given survey can be fetched using:
+
++ Parameters
+
+    + survey_id (integer, required) ... the Survey ID
+    + businessNumber (string, required) ... the Business Number of Enterprise (for associated api key)
+    + api_key (string, required) ... the API key
+
+### List Responses for a Survey [POST]
+
+<center><b>Example curl request</b></center>
+
+    curl -H "content-type: application/json" -H "accept:application/json" -X POST -d "{}"  "https://api.birdeye.com/resources/v1/survey/list/responses/1891?page=0&size=3&sortby=date&sorder=1&api_key=56eSPYy0zNxKWltvTziIJiebx0yU9pVT&businessNumber=146611277075427" -v
```

Comment:
@sachmuchsachin Is the API key used here a valid API key? Replace it with some fake API key like kjsafjasflal89q2eksjfasf